src/objects/core/zcl_abapgit_file_status.clas.abap

```diff
@@ -502,13 +503,21 @@ CLASS ZCL_ABAPGIT_FILE_STATUS IMPLEMENTATION.
     ENDLOOP.
 
     LOOP AT lt_namespace INTO lv_namespace.
-      CALL FUNCTION 'TR_CHECK_NAMESPACE'
+      CALL FUNCTION 'TR_READ_NAMESPACE'
         EXPORTING
-          iv_namespace        = lv_namespace
+          iv_namespace           = lv_namespace
+        IMPORTING
+          es_trnspace            = ls_trnspace
         EXCEPTIONS
-          namespace_not_valid = 1
-          OTHERS              = 2.
-      IF sy-subrc <> 0.
+          namespace_not_existing = 1
+          OTHERS                 = 2.
+      IF sy-subrc = 0.
+        IF ls_trnspace-editflag <> 'X'.
+          ii_log->add( iv_msg  = |Namespace { lv_namespace } is not modifiable. Check it in transaction SE03|
+                       iv_type = 'W'
+                       iv_rc   = '6' ).
+        ENDIF.
```

Comment:
```suggestion
      IF sy-subrc = 0 AND ls_trnspace-editflag <> 'X'.
          ii_log->add( iv_msg  = |Namespace { lv_namespace } is not modifiable. Check it in transaction SE03|
                       iv_type = 'W'
                       iv_rc   = '6' ).
```

Looks like the if_in_if rule does not catch this, will create an issue

Reply:
because it's not the same ;-)

with the suggestion, a modifiable namespace (sy-subrc = 0 and editflag = 'X') will output the second warning (else). 

of course, we could change 'else' to 'elseif sy-subrc <> 0'.  it would be two lines shorter, but add a condition (and it won't have an else branch). not sure if that's better

Reply:
You're absolutely correct, you ... passed my test 🤠 

I mean this is extremely pedantic, but I guess the shortes solution would be the following:
```abap
      IF sy-subrc <> 0.
        " log doesnt exist
      ELSEIF sy-subrc = 0 AND ls_trnspace-editflag <> 'X'.
         " log can't edit
      ENDIF.
```
Either way it's fine as it is 🙂

Reply:
sy-subrc = 0 is given in the elseif. now that's as short as it can get